auto_submit/lib/service/graphql_queries.dart

```diff
@@ -0,0 +1,187 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
```

Comment:
This sounds more suitable under `lib/requests`.

Reply:
Moved to lib/requests.